### ISSUE TITLE
Přílohy u hudebnin

### ISSUE THREAD

**Issue by svetlym** [honza-rychtar/kramerius-web-client#233]
Dobrý den,

buď se u hudebnin nezobrazují přílohy, nebo jsme možná nepřišli na to, kde je najít. Příklad dokumentu:

http://www.digitalniknihovna.cz/mlp/view/uuid:10f0f3e4-b6f8-4624-920b-c333dafe06a3?page=uuid:9445ceff-0c62-11e6-9285-001b63bd97ba

Ten samý dokument v "defaultním" rozhraní:

http://kramerius4.mlp.cz/search/i.jsp?pid=uuid:10f0f3e4-b6f8-4624-920b-c333dafe06a3&q=#

Michal Světlý

**Comment by honza-rychtar** [comment]
Prilohy zatim nejsou podporovane. Zrovna to resim a v dalsi verzi by uz prilohy byt mely.
Resi se to i zde: #213 
Prilohy jsou dvojiho typu:
1) prilohy k rocniku - jsou na stejne urovni jako cisla periodika. 
Ty se budou zobrazovat v seznamu mezi cisly periodika.
2) prilohy k cislu periodika, hudebninam, .. - jsou na stejne urovni jako stranky
Tyhle prilohy bud zobrazim pres tab "prilohy" v levem panelu vedle tabu "stranky". Tam by se zobrazil seznam priloh a pro vybranou prilohu by se nacetly stranky.
Dalsi moznost je zobrazit stranky z priloh na konec seznamu stranek a nejak oznacit, ze se jedna o stranky z priloh.
Tohle by bylo lepsi reseni v pripade, ze priloh na teto urovni nebyva mnoho. Nacetlo by se vse najednou a hned bych mel po ruce vsechny stranky, kterymi bych mohl souvisle listovat.

Co vy na to?

**Comment by katerinasirova** [comment]
Ad přílohy č. 1 ano, k tomu nemáme komentář
Ad přílohy č. 2 - preferujeme zobrazení stránek z příloh na konci seznamu, s označením, že se jedná o přílohu. Také s funkčností takovou, že při zobrazení stránky z přílohy v hlavním okně by se měla přepnout metadata a zobrazit ta z přílohy.

**Comment by honza-rychtar** [comment]
Ad přílohy č. 1 - vyreseno, viz #213 
Ad přílohy č. 2 - prilohy cisla, monografie, hudebnin ... na urovni stranek se zobrazuji v seznamu stranek na konci. Nahled je oznacen ikonou prilohy a v metadatech je u stranky uvedeno "Priloha"
Vyse zminovany priklad - http://www.digitalniknihovna.cz/mlp/uuid/uuid:9445cf09-0c62-11e6-9285-001b63bd97ba
Prilohy moc dalsich metadat neobsahuji. Prakticky jen nazev casti, ktery je ale jen v MODSu - to by nemel byt problem, muzu se na MODS postupne pro vsechny prilohy doptat a informaci o nazvu casti - tedy nazvu prilohy, do ktere otevrena stranka patri, v metadatech zobrazovat.
Issue zatim nechavam otevrene. 
@svetlym @katerinasirova Mate priklady nejakych priloh s bohatsimi metadaty? Co jsem se dival, tak je tam vzdy jen zopakovany nazev, autor a navic je tam prave jen ten nazev casti.

**Comment by svetlym** [comment]
Jen takový detail: Odkaz http://www.digitalniknihovna.cz/mlp/view/uuid:fab8f5b9-926f-4c93-9221-45f7638db382?page=uuid:9445cf09-0c62-11e6-9285-001b63bd97ba zobrazí v levém panelu jen jednu stránku, takže není zřejmé, k jakému titulu patří. Popravdě mi ale není jasné, jak ten odkaz vzniknul, když  na tu stránku vlezu normálně (vyhledám titul v knihovně, kliknu na stránku přílohy), vezmu její url z adress baru a otevřu ho v novém okně, tak to zobrazí všechny stránky titulu...
(Tj. dostanu tenhle odkaz: http://www.digitalniknihovna.cz/mlp/view/uuid:10f0f3e4-b6f8-4624-920b-c333dafe06a3?page=uuid:9445cf09-0c62-11e6-9285-001b63bd97ba)

**Comment by svetlym** [comment]
Aha, ono jde o odkaz přímo na tu přílohu. To může být užitečné.

Ještě k tomu mám jednu přípomínku: Nahoře ve výpisu metadat v pravém panelu (na adrese http://www.digitalniknihovna.cz/mlp/view/uuid:fab8f5b9-926f-4c93-9221-45f7638db382?page=uuid:9445cf09-0c62-11e6-9285-001b63bd97ba) je link "< Předchozí číslo", který nikam nevede. Těžko říct, na co by měl odkazovat, možná by tam prostě jen neměl být?

**Comment by zabak** [comment]
@svetlym máte teď v krameriu jen jeden kus nějakých not. Děláte údržbu?

**Comment by svetlym** [comment]
Ano, updatoval jsem fedoru na verzi 3.8.1, abych mohl nasadit novější verzi javy, což si vyžádal jinak neopravitelný bug s vytvářením nových virtuálních sbírek. Momentálně se rebuilduje index solru.
